### PR TITLE
On boarding new operator from Konveyor

### PR DESCRIPTION
**Pull request by JaydipGabani**

### CODE REVIEW

File: ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  mig-operator:
```
Comment (shawn-hurley) [inline ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml:6]: I don't think we need a base of mig-operator, we may need a base of ansible-operator though

File: ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    name: "4.8"
+    namespace: ocp
+    tag: mig-operator
+binary_build_commands: make build
+build_root:
+  use_build_cache: true
+  project_image:
+    dockerfile_path: build/Dockerfile
+images:
+  - dockerfile_path: build/Dockerfile
```
Comment (shawn-hurley) [inline ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml:16]: Why are we building two images from the same dockerfile with different bases?

**Review by shawn-hurley — CHANGES_REQUESTED** [review]
A good start some minor stuff that we should fix

File: ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml (hunk 2 above)
Comment (JaydipGabani) [reply ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml:16]: Ahh, removng it

File: ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+base_images:
+  base:
+    name: base
+    namespace: ocp
+    tag: latest
+  ansible-operator:
+    name: anisble-operator
+    namespace: ocp
+    tag: mig-operator
```
Comment (shawn-hurley) [inline ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml:9]: I think that you want this tagged as ansible operator and mig-operator to be the image we build

File: core-services/prow/02_config/_plugins.yaml [hunk 4]
```diff
@@ -5058,6 +5079,34 @@ plugins:
     - verify-owners
     - wip
     - yuks
+  konveyor/mig-operator:
+    plugins:
+    - assign
+    - blunderbuss
+    - bugzilla
```
Comment (shawn-hurley) [inline core-services/prow/02_config/_plugins.yaml:5086]: I don't think we want the bugzilla plugin

Comment (shawn-hurley) [reply core-services/prow/02_config/_plugins.yaml:5086]: There are bunch of plugins here, that are really usefull for the merge bot, but I don't think that we ae using the merge bot right? Things like jira, lgtm/hold/override/wip/approve  I don't think are needed.

**Review by shawn-hurley — CHANGES_REQUESTED** [review]

File: core-services/prow/02_config/_plugins.yaml (hunk 4 above)
Comment (JaydipGabani) [reply core-services/prow/02_config/_plugins.yaml:5086]: +1 you are on the mark! I will remove them

File: ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml (hunk 3 above)
Comment (JaydipGabani) [reply ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml:9]: +1

File: ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
+base_images:
+  ansible-operator:
+    name: anisble-operator
```
Comment (shawn-hurley) [inline ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml:3]: s/anisble-operator/ansible-operator

Comment (JaydipGabani) [reply ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml]: Thank you! Missed it

File: ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml [hunk 6]
```diff
@@ -0,0 +1,56 @@
+base_images:
+  ansible-operator:
+    name: ansible-operator
```
Comment (shawn-hurley) [inline ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml:3]: Pretty sure that this needs to be

```
  name: "4.9"
  namespace: ocp
  tag: ansible-operator

**Review by shawn-hurley — APPROVED** [review]

**Comment by AlexNPavel** [comment]
Issue I found looking through the pod logs. The migration operator fails to start with this error:

```
/usr/local/bin/entrypoint: line 20: /tini: No such file or directory
```

**Comment by shawn-hurley** [comment]
@AlexNPavel I wonder if there is a way to determine that the operator pod is failing to come up and add that to the logs? I think you are correct that the operator is failing because of our entry point and we are fixing that sorry about the noise in test platform.

**Pull request merged**